src/lsp-server.ts

```diff
@@ -1308,6 +1312,24 @@ export class LspServer {
             params.textDocument.uri, params.range, this.documents, this.tspClient, this.options.lspClient, this.configurationManager, token);
     }
 
+    async linkedEditingRange(params: lsp.LinkedEditingRangeParams, token?: lsp.CancellationToken): Promise<lsp.LinkedEditingRanges> {
```

Comment:
I see that the lsp libraries define the return value as `LinkedEditingRanges | null`. What happens if we return `null`?

Reply:
Yes, I missed that.
I pushed a commit to return `null` in cases when ts can't compute LinkedEditingRanges.